### ISSUE TITLE
narrator scan mode [ctrl+end] crashes conhost

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#7839]
_ut supra_

**Comment by DHowett** [comment]
```
09 000000ae`3f4fe7a0 00007ffc`6495acd0     conhost!Microsoft::Console::Types::UiaTextRangeBase::ScrollIntoView+0x2a1 [onecore\windows\core\console\open\src\types\uiatextrangebase.cpp @ 813] 
```

```

    FAIL_FAST_IF(!(newViewport.Top >= topRow));
```

```
0:009> dv newViewport
    newViewport = {LT(0, 8962) RB(119, 9001) In:[120 x 40] Ex:[119 x 39]}
```

top=8962

in Vb conhost, ctrl+end moves to the ... top of the buffer :|

**Comment by DHowett** [comment]
Oh, it's bottom that's wrong here

**Comment by DHowett** [comment]
```
    [+0x048] _start           : {0,9000} [Type: _COORD]
    [+0x04c] _end             : {0,9001} [Type: _COORD]
```
